### ISSUE TITLE
Remove supportsNamedVirtualRegisters from Z CodeGenerator

### ISSUE THREAD

**Issue by 0xdaryl** [eclipse/omr#2059]
The function `OMR::Z::CodeGenerator::supportsNamedVirtualRegisters` is not used in OMR or known downstream projects.  Delete it.

**Comment by nanjekyejoannah** [comment]
@0xdaryl shouldnt this issue be closed? Looks like the PR #2118 fixed this. If anything is missing, I could look at it.

**Comment by 0xdaryl** [comment]
Yes.  I'm not sure why it did not close automatically.  Possibly because there were multiple issues listed.  Thanks for pointing that out.  Closing...
